compiler-plugin/src/main/java/io/ballerina/stdlib/kafka/plugin/PluginUtils.java

```diff
@@ -73,4 +79,17 @@ public static boolean validateModuleId(ModuleSymbol moduleSymbol) {
         }
         return false;
     }
+
+
```

Comment:
```suggestion
```

Reply:
Let's remove these 2 new lines.

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-kafka/pull/368/commits/c8a0e3f012589077bc84147b587b3b69faac8d64